store/helper/helper.go

```diff
@@ -149,13 +149,26 @@ func (h *Helper) FetchHotRegion(rw string) (map[uint64]RegionMetric, error) {
 		return nil, errors.Trace(err)
 	}
 	metricCnt := 0
+	regionCnt := make(map[uint64]int)
 	for _, hotRegions := range regionResp.AsLeader {
 		metricCnt += len(hotRegions.RegionsStat)
+		for _, region := range hotRegions.RegionsStat {
+			regionCnt[region.RegionID]++
+		}
+	}
+	for _, hotRegions := range regionResp.AsPeer {
+		for _, region := range hotRegions.RegionsStat {
+			regionCnt[region.RegionID]++
+		}
 	}
 	metric := make(map[uint64]RegionMetric, metricCnt)
 	for _, hotRegions := range regionResp.AsLeader {
 		for _, region := range hotRegions.RegionsStat {
-			metric[region.RegionID] = RegionMetric{FlowBytes: uint64(region.FlowBytes), MaxHotDegree: region.HotDegree}
+			metric[region.RegionID] = RegionMetric{
+				FlowBytes:    uint64(region.FlowBytes),
+				MaxHotDegree: region.HotDegree,
+				Count:        regionCnt[region.RegionID],
```

Comment:
According to https://docs.pingcap.com/tidb/stable/information-schema-tidb-hot-regions#tidb_hot_regions, if I understand correctly, the `Count` here is the total number of regions in the store which contains this hot region, instead of the number of the replicas of this hot region?

Reply:
my bad... will fix it soon

Reply:
Sorry to delay mant time... but I have no idea about how to get the number of regions in the store, I think use pd api will make the latency too serious? @eurekaka